charts/bar-api/values.preview.template.yaml

```diff
@@ -5,3 +5,5 @@ java:
     S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     IDAM_CLIENT_BASE_URL: https://idam-api.aat.platform.hmcts.net
     PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
+  postgresql:
```

Comment:
Indentation should be like here : https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.preview.template.yaml#L13